flink-core/src/main/java/org/apache/flink/configuration/RestOptions.java

```diff
@@ -33,7 +33,7 @@
 	 */
 	public static final ConfigOption<String> REST_ADDRESS =
 		key("rest.address")
-			.defaultValue("localhost")
+			.defaultValue("0.0.0.0")
```

Comment:
It's not enough to set the default value to `0.0.0.0`. The rest client uses the value under this config key to connect. I am taking over this ticket now. #5707

Reply:
Can I close this PR?

Reply:
yes you can close it